src/main/java/de/themorpheus/edu/taskservice/database/repository/solution/SolutionFreestyleRepository.java

```diff
@@ -0,0 +1,10 @@
+package de.themorpheus.edu.taskservice.database.repository.solution;
```

Comment:
Please also rename all Repository classes to GenericSolutionRepository

Reply:
How? SolutionRepository = GenericSolutionRepository?

Reply:
Generic stands for Specific type...
so FreestyleSolutionRepository
or WordsaladSolutionRepository